lib/registries/addon/components/license-viewer/styles.scss

```diff
@@ -2,3 +2,11 @@
     padding: 0;
     border: 0;
 }
+
+.LicenseView {
+    overflow: auto;
```

Comment:
I think we wanted a `text-overflow: ellipsis` for this as well, based on Product's feedback.

Reply:
Of course. I am creating that change now.